src/PAModel/PAConvert/Yaml/YamlWriter.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.PowerPlatform.Formulas.Tools.Yaml
     /// Notably, property values are always either multi-line escaped or
     /// prefixed with '=' to block yaml from treating it as a yaml expression.
     /// </summary>
-    internal class YamlWriter
+    public class YamlWriter
     {
```

Comment:
Double checking, is there anything in here that becomes public that we don’t need to be?

Reply:
The class itself isn't tied to anything else and everything else in the same namespace is marked `internal`. I want everything that is already public on it to be available, so I think we're all good here.

Reply:
Ah you know what - `NormalizeNewlines` is only used within this class. Let me mark that as private, it doesn't need to be exposed.

Reply:
Okay, changed that to private and removed the static designation as it didn't seem necessary.